### PR TITLE
DOC: Fix the docstring of `write_mapping`

### LINKED ISSUE #2567: This should be a `1`
Looking through this code, I noticed that the docstring is wrong. Posting this issue so that I don't forget to come back and fix it.

https://github.com/dipy/dipy/blob/3687ce9662280869d27480f7752ad663fefc6b28/dipy/align/_public.py#L238

### PR DESCRIPTION
**Pull request by arokem**
Closes #2567.

### CODE REVIEW

File: dipy/align/_public.py [hunk 1]
```diff
@@ -233,9 +233,9 @@ def write_mapping(mapping, fname):
 
     Notes
     -----
-    The data in the file is organized with shape (X, Y, Z, 2, 3, 3), such
-    that the forward mapping in each voxel is in `data[i, j, k, 0, :, :]` and
-    the backward mapping in each voxel is in `data[i, j, k, 0, :, :]`.
+    The data in the file is organized with shape (X, Y, Z, 3, 2), such
+    that the forward mapping in each voxel is in `data[i, j, k, :, 0]` and
```
Comment (skoudoro) [inline dipy/align/_public.py:237]: should not be `data[i, j, k, 3, 0]` instead since you define 3 above?

Comment (skoudoro) [reply dipy/align/_public.py:237]: I mean `3` instead of `:`

Comment (arokem) [reply dipy/align/_public.py:237]: I might be misunderstanding your question, but there are 3 components to the mapping (dx, dy, dz) so the full forward mapping is data[:, :, :, :, 0] and has dimensions `(X,Y,Z,3)`

Comment (skoudoro) [reply dipy/align/_public.py:237]: yes, got it, I should go to sleep... :-)

**Comment by skoudoro** [comment]
Failing CI's not related, merging

**Pull request merged**